Library/Formula/go.rb

```diff
@@ -76,9 +93,10 @@ def caveats; <<-EOS.undent
     `go vet` and `go doc` are now part of the go.tools sub repo:
       http://golang.org/doc/go1.2#go_tools_godoc
 
-    To get `go vet` and `go doc` run:
+    `godoc` has been installed for you unless you requested otherwise.
+
+    To get `go vet` run:
       go get golang.org/x/tools/cmd/vet
-      go get golang.org/x/tools/cmd/godoc
```

Comment:
Sure, It's easy enough to do now we have the resource in place. We can just use the resource twice.

> we know if they did by

I didn't know if it was worth splitting up the caveats for this situation, to cater to the appropriate ` if build.with? ` situation, but I'm happy to do so if you want me to?